### PR TITLE
Fix worker crash vulnerability - Closes #1081

### LINKED ISSUE #1081: Worker reboot issue in Lisk 1.0
Right now it's possible for a peer to reboot another peer's worker process if they send a specially crafted message (because it triggers a fatal error on the instance on the target). This issue affects SocketCluster v8 and earlier (including v5 which Lisk is using).

The simplest/cleanest solution would be to upgrade to SC v9. Upgrade from v5 should be relatively straight forward.

### PR DESCRIPTION
**Pull request by jondubois**
Closes #1081

Note that a big portion of the diff lines below are caused by indentation change.

### CODE REVIEW

File: workersController.js [hunk 1]
```diff
@@ -19,94 +20,96 @@ var config = require('./config.json');
  * Function is invoked by SocketCluster
  * @param {Worker} worker
  */
-module.exports.run = function (worker) {
+class Worker extends SCWorker {
```
Comment (MaciejBaj) [inline workersController.js]: The whole codebase is currently in ES5 code standards, so a user can still run Lisk from source on any Node.js version. Is it possible to set up `SCWorker` in a different way than extending a class?

Comment (jondubois) [reply workersController.js]: @MaciejBaj Yes it's possible, I'll change this.

File: workersController.js [hunk 2]
```diff
@@ -16,97 +17,98 @@ var Logger = require('./logger');
...
+ *  is ready to accept requests/connections.
  */
-module.exports.run = function (worker) {
+SCWorker.create({
+	run: function () {
+		var self = this;
+		var scServer = this.getSCServer();
 
-	var scServer = worker.getSCServer();
+		async.auto({
```
Comment (MaciejBaj) [inline workersController.js]: Unnecessary empty line just below `async.auto({`.

**Comment by MaciejBaj** [comment]
Congratulations on the first contribution ;)

**Comment by karmacoma** [comment]
@jondubois Upon running Jenkins we have the following failures:

```
RPC
    internal
  handshake
      updatePeer
        schema
    with invalid headers
      should fail with INVALID_HEADERS code and description
        ✓ without headers (59ms)
        ✓ with empty headers
        ✓ without port
  WS transport blocks
    blocks
        ✓ without height
        ✓ without version
        ✓ without nethash
    with valid headers
      1) "before all" hook
      2) "after all" hook


  6 passing (3s)
  2 failing

  1) handshake
       with valid headers
         "before all" hook:
     TypeError: SocketCluster is not a constructor
      at Object.start (test/common/ws/server.js:20:28)
      at Context.<anonymous> (test/functional/ws/transport.handshake.js:162:13)

  2) handshake
       with valid headers
         "after all" hook:
     Error: No SocketCluster instance running
      at Object.stop (test/common/ws/server.js:25:10)
      at Context.<anonymous> (test/functional/ws/transport.handshake.js:166:13)
```

**Comment by jondubois** [comment]
@karmacoma @MaciejBaj The tests should pass once https://github.com/LiskHQ/wamp-socket-cluster/pull/23 is merged and we bump up the version number for `wamp-socket-cluster` on this repo.

**Pull request merged**